src/renderer/base/RenderSettings.cpp

```diff
@@ -187,6 +187,7 @@ std::pair<COLORREF, COLORREF> RenderSettings::GetAttributeColors(const TextAttri
     // We want to nudge the foreground color to make it more perceivable only for the
     // default color pairs within the color table
     if (GetRenderMode(Mode::DistinguishableColors) &&
+        Feature_AdjustIndistinguishableText::IsEnabled() &&

```

Comment:
I believe these two lines should be swapped.

In practice it really won't make a difference here (because a compiler _should_ be smart enough to figure out that `IsEnabled()` equals `false` and that the entire if condition can be optimized away), but in theory at least there's no guarantee that the compiler will always be smart enough to optimize it away. Additionally this only works well if the previous call(s) are inlineable and have no side effects (allowing it to be optimized away).

That's why we should try to move conditions that have the highest chance of being predictable to the beginning of a boolean expression.

Reply:
Good to know! Changed it, thanks